package.json

```diff
@@ -33,7 +33,13 @@
     "resolve-from": "^1.0.0"
   },
   "devDependencies": {
-    "ava": "0.0.3",
+    "ava": "*",
+    "xo": "^0.16.0",
     "heapdump": "^0.3.7"
+  },
+  "xo": {
+    "rules": {
+      "import/no-dynamic-require": "off"
```

Comment:
It's best practice to disable rules on a per line basis instead of globally by adding `// eslint-disable-line import/no-dynamic-require` after the `heapdump.writeSnapshot` line.